### PR TITLE
v1.5.0 release

### PR DESCRIPTION
**Pull request by localvar**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,40 @@
 # Changelog
 
+## [v1.5.0](https://github.com/megaease/easegress/tree/v1.5.0) (2022-03-03)
+
+[Full Changelog](https://github.com/megaease/easegress/compare/v1.4.1...v1.5.0)
+
```
Comment (xxx7xxxx) [inline CHANGELOG.md:6]: Won't we deprecate the dynamic cluster and warn to delete it in the next release?

Comment (localvar) [reply CHANGELOG.md:6]: Oh, forgot this, will update now.

Comment (localvar) [reply CHANGELOG.md:6]: updated.

**Review by xxx7xxxx — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,42 @@
...
+**Significant changes:**
+
+- HTTP basic auth filter [\#454](https://github.com/megaease/easegress/pull/454)
+- HeaderLookup filter [\#454](https://github.com/megaease/easegress/pull/454)
+- HeaderToJSON filter [\#458](https://github.com/megaease/easegress/pull/458)
+- CertExtractor filter [\#474](https://github.com/megaease/easegress/pull/474)
+- Custom data management [\#456](https://github.com/megaease/easegress/pull/456), [\#500](https://github.com/megaease/easegress/pull/500)
+
+  **NOTE:** The dynamic cluster management feature (e.g. adding a new primary node without stopping the cluster) is deprecated and will be removed in the next release, please switch to static cluster management.
```
Comment (samutamm) [inline CHANGELOG.md:15]:
```suggestion
  **NOTE:** The dynamic cluster management feature (e.g. adding a new primary node without stopping the cluster) is deprecated and will be removed in the next release, please switch to static cluster management. You can refer this cookbook [chapter](https://github.com/megaease/easegress/blob/main/doc/cookbook/multi-node-cluster.md) or follow [Helm example](https://github.com/megaease/easegress/tree/main/helm-charts/easegress) for more info, how to define static cluster.
```

**Review by samutamm — APPROVED** [review]

**Review by suchen-sci — APPROVED** [review]

**Pull request merged**